Source/SGDLib/SGD.cpp

```diff
@@ -3165,6 +3165,9 @@ SGDParams::SGDParams(const ConfigRecordType& configSGD, size_t sizeofElemType)
             m_enableDistributedMBReadingNotSpecified = !configParallelTrain.Exists(L"distributedMBReading");
             m_enableDistributedMBReading = configParallelTrain(L"distributedMBReading", false);
             m_syncStatsTrace = configParallelTrain(L"syncPerfStats", (int)0);
+            bool useV2Aggregator = configParallelTrain(L"useV2Aggregator", false);
```

Comment:
Is there any test case in CNTK code base that has this configuration set to true?